packages/web-client/app/components/card-pay/balance-display/index.hbs

```diff
@@ -22,7 +22,7 @@
         </div>
         {{#if @usdBalance}}
           <div class="balance-display__text" data-test-balance-display-usd-amount>
-            ${{@usdBalance}} USD
+            ${{@usdBalance}} USD{{if @usdFootnote "*"}}
```

Comment:
Afaik the disclaimer is always true. USD balance is an estimate. I think there was a designwide decision on how this is presented - with/without the *, using the estimate (≈) symbol or not. @burieberry  do you remember what it is, if there was one?

Reply:
I agree with the comments about the complexity of this card.

In terms of the approximate sign (≈), in a design meeting we said that we won't use it. (I had to use it in one place though (the face value selection card in the issue card workflow) because the DAI amounts were very rounded.) The footnotes remain as is.